spec/jobs/rapid_ready_for_decision/form526_asthma_job_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sidekiq/testing'
+
+RSpec.describe RapidReadyForDecision::Form526AsthmaJob, type: :worker do
+  subject { described_class }
+
+  before do
+    Sidekiq::Worker.clear_all
+  end
+
+  let!(:user) { FactoryBot.create(:disabilities_compensation_user, icn: '2000163') }
+  let(:auth_headers) do
+    EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h)
+  end
+  let(:saved_claim) { FactoryBot.create(:va526ez) }
+  let(:submission) do
+    create(:form526_submission, :with_uploads,
+           user_uuid: user.uuid,
+           auth_headers_json: auth_headers.to_json,
+           saved_claim_id: saved_claim.id,
+           submitted_claim_id: '600130094')
+  end
```

Comment:
Try with this simpler setup:
```suggestion
  let(:submission) do
    create(:form526_submission, :with_uploads,
           saved_claim_id: saved_claim.id,
           submitted_claim_id: '600130094')
  end
```
or
```suggestion
  let(:submission) do
    create(:form526_submission, :with_uploads)
  end
```